packages/suite/src/hooks/wallet/coinmarket/form/useCoinmarketVerifyAccount.tsx

```diff
@@ -129,7 +130,9 @@ const useCoinmarketVerifyAccount = ({
     );
 
     const { supportedMainnets, supportedTestnets } = useNetworkSupport();
-    const findNetworkByType = (network: Network) => network.networkType === currency;
+    const currencyNetwork = currency === undefined ? undefined : cryptoIdToNetwork(currency);
```

Comment:
Nitpick: Could you please change the property name from `currency` to `cryptoId`. It would be beneficial next time.